backend/backends/null/null.go

```diff
@@ -0,0 +1,39 @@
+package null
+
+import (
+	"github.com/atlassian/gostatsd/backend"
+	"github.com/atlassian/gostatsd/types"
+
+	"github.com/spf13/viper"
+)
+
+const backendName = "null"
+
+func init() {
```

Comment:
The program reads from bottom to top, so you put the `init()` function at the bottom and then the next function in the stack just above, etc. Again don't need to change and I haven't done it consistently, look at the datadog backend for an example.